crates/forge/bin/cmd/script/transaction.rs

```diff
@@ -410,7 +385,7 @@ pub mod wrapper {
         #[serde(rename = "gasUsed")]
         pub gas_used: Option<U256>,
         /// Contract address created, or `None` if not a deployment.
-        #[serde(rename = "contractAddress", serialize_with = "serialize_opt_addr")]
```

Comment:
do we not need this?

Reply:
serialize fns don't do anything different than default impl

Reply:
default impl does checksum? if so, all good, think this was just leftover from ethers

Reply:
Ah you're right, we wrapped to checksum, I don't know why but I'll keep it